### DISCUSSION TITLE
New to PhotoPrism, why won't all my photos in my Synology NAS get added to library?

### DISCUSSION THREAD

**Discussion by altaheer** [photoprism/photoprism#3349] [Q&A]
I have installed Photoprism and it works fine except one important thing.
Ive added my Synology NAS photos folder to the the directories shows under All Originals drop down menu but it wont actually add all the images in that folder.

**Comment by lastzero** [comment]
There can be many reasons for this, such as incorrect file system permissions, unsupported file extensions, or a file size limit you may have configured:

👉 https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Reply by scubieman** [reply]
How can we test to see which one of them makes it fail? I have 780 photos of over 1k that loaded in just fine.
